frontend/src/Components/Login/LoginForm.js

```diff
@@ -45,7 +45,7 @@ const LoginForm = ({ mode }) => {
 
     try {
       if (mode === 'login') {
-        const response = await axios.post('http://localhost:5000/login', { email: username, password });
```

Comment:
is the backend deployed wroking properly for now?

Reply:
yasss